serverless/deploy/docker/witness/README.md

```diff
@@ -0,0 +1,32 @@
+Witness
+-------
+
+This [docker-compose.yaml](docker-compose.yaml) file can be used to spin up a witness daemon.
+
+You should set the following environmemt variables (either `export` or with a `.env` file):
```

Comment:
I would split this into two sections: required, and optional.

I got stuck on the first entry wondering why it was the most important thing, when it's an internal config _within_ the docker image. It's not even on my filesystem. Oh, it has a default. Am I supposed to put things in there? It took me longer than it should to realize I probably don't need to fiddle with this.

Looking at it again, I think ordering the top 3 as (key, dir, file) and then tacking them on the bottom may be enough?

Reply:
done